docs/index.md

```diff
@@ -107,11 +107,11 @@ Versions in Linux distributions:
 * [1.4.2](Release-Notes-1.4.2) (2017-06-20)
 * [1.4.1](Release-Notes-1.4.1) (2017-04-26)
 * [1.3.5](Release-Notes-1.3.5) - only for EL6
-* [1.3.4](Release-Notes-1.3.4)
-* [1.3.3](Release-Notes-1.3.3) (2017-01-01) - bugfixing; a new config option for the builder hostname; upgraded temporary directories, chroot contains `best=1`
+* [1.3.4](Release-Notes-1.3.4) - new extension for .log `available_pkgs.log` and `installed_pkgs.log`, new configuration files, bugfixing
```

Comment:
Can you add (2017-02-27) becaue `AuthorDate: Mon Feb 27 17:26:04 2017 +0100`

Reply:
I added the date.